CRM/Mailing/BAO/Mailing.php

```diff
@@ -3197,4 +3197,51 @@ public static function getPublicViewUrl($id, $absolute = TRUE) {
     }
   }
 
+  /**
+   * @return array
+   *   A list of template-types, keyed by name. Each defines:
```

Comment:
I think this is incorrect. It is keyed by an arbitrary weight-based number. The alternate function is keyed by name.

Also, I think our comment standard (the full one not the jenkins enforced one) is to have a sentence before the params (cap letter, full stop) that makes ides all warm & fuzzy